ecl/hthor/hthor.cpp

```diff
@@ -2335,7 +2335,7 @@ const void *CHThorRollupActivity::nextRow()
 
 //=====================================================================================================
 
-CHThorGroupDedupActivity::CHThorGroupDedupActivity(IAgentContext &_agent, unsigned _activityId, unsigned _subgraphId, IHThorDedupArg &_arg, ThorActivityKind _kind) : CHThorSimpleActivityBase(_agent, _activityId, _subgraphId, _arg, _kind), helper(_arg)
+CHThorGroupDedupActivity::CHThorGroupDedupActivity(IAgentContext &_agent, unsigned _activityId, unsigned _subgraphId, IHThorDedupArg &_arg, ThorActivityKind _kind) : CHThorSimpleActivityBase(_agent, _activityId, _subgraphId, _arg, _kind), helper(_arg), compareBest(0)
```

Comment:
If the CHThorGroupDedupKeepRightActivity activity was always used when best is selected this code wouldn't need to change.